docs/help/operator.md

```diff
@@ -1029,3 +1029,104 @@ stringData:
   jolokiaUser: alice
   jolokiaPassword: password1
 ```
+
+## Configuring Additional Volumes to the Broker
+
+### Attaching extra volumes shared by all broker pods
+
+When you have some existing volumes to be used by brokers, you can use **spec.deploymentPlan.extraVolumes** and **spec.deploymentPlan.extraVolumeMounts** to configure a broker custom resource to mount those volumes. For example if you have a PersistentVolumeClaim type volume called **my-pvc-0** and it has 1G capacity with some useful data in it:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: my-pvc-0
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+```
+You can configure a broker CR to use it:
+
+```yaml
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: artemis-broker
+spec:
+  deploymentPlan:
+    size: 2
+    extraVolumes:
+      - name: mydata
+        persistentVolumeClaim:
+          claimName: my-pvc-0
+    extraVolumeMounts:
+      - name: mydata
+        mountPath: /opt/mydata
+```
+When deploying the above CR, the PVC volume will be mounted to path **/opt/mydata** in the broker container of both broker pods. The **extraVolumeMounts** is optional. If not specified a default mountPath is given based on the type of the volume, as shown in the following list:
+
+|type of volume|default mountPath|
+|---|---|
+|HostPath|/amq/extra/hostpaths/<volume.name>|
```

Comment:
this must be a pattern that we can describe, otherwise this list will be out of date very quickly. Just describing the pattern should be sufficient.

Reply:
It makes sense. will update. Thanks!